presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/RandomizeNullKeyInOuterJoin.java

```diff
@@ -366,6 +374,7 @@ public PlanNode visitJoin(JoinNode joinNode, RewriteContext<Set<VariableReferenc
                             Optional.empty(),
```

Comment:
should we also remove these inlined calls? This should now be properly handled outside in the main optimizer loop

Reply:
Sure, let me remove this code too.